slam/curvature.py

```diff
@@ -87,18 +87,18 @@ def curvature_fit(mesh, tol=1e-12, neighbour_size=2):
 
 def determine_local_basis(normal, tol):
     """
-    Computes the local basis of the tangent plane
-    :param normal: (3,1) normal vector
+    Computes the orthonormal local basis (vec1, vec2) of the tangent plane
+    :param normal: (3,1) normal vector, should be unitary
     :param tol: tolerance value
     :return:
```

Comment:
please complete the docstring with the description of the ouput vectors

Reply:
I added some comments for the return. It is a good thing to make the code the more explicit as possible, but "explicit" has also to be explicit (please add standards/rules somewhere in order to know exactly what to do in a PR, in commenting/docstringing the code etc).

Reply:
Nice ty @JulienLefevreMars I am currently trying to add modifications to your PR but i am still struggling for the push. Hope i l solve it soon

Reply:
it's great @alexpron , it provides a good canvas for how to write a proper docstring, thks !